package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "eslint.github.io",
+  "version": "1.0.0",
+  "description": "This contains the code running on eslint.org.",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "build": "webpack --module-bind json js/app/parser/index.js js/app/parser/index.built.js",
```

Comment:
In that case, do you want to remove index.build.js and webpack script and just use eslint.js instead?